### ISSUE TITLE
Dark mode not supported (Edge on iOS)

### ISSUE THREAD

**Issue by rachaelpenfil** [department-of-veterans-affairs/vets-design-system-documentation#2744]
# Bug Report

- [x] I’ve searched for any related issues and avoided creating a duplicate issue.

## What happened

When using the Edge browser on iOS in dark mode, most of the content isn't readable and there are lots of weird visual bugs, including a white box around the stepper. 

![screenshot of name and date of birth page in dark mode on iOS in Edge. Text is dark gray on a black background and unreadable](https://github.com/department-of-veterans-affairs/vets-design-system-documentation/assets/147765613/a5e8bd3f-14d6-488f-a204-327b13ca5450)

The issue does not exist in Chrome or Safari.

## What I expected to happen

Colors should invert so a11y contrast ratios are met. We should support dark mode (not necessarily through a separate colorway) on iOS devices, which is also something we could contribute back to USWDS. @eli-oat and @briandeconinck may be able to provide additional detail here.

## Reproducing

- Formation version:
- Device: iOS (tested on iPhone 13)
- Browser: Edge

Steps to reproduce:

1. Device should be in dark mode. Settings > Display & Brightness > Dark
2. Open a new instance in Edge - issue may not reproduce if you're in an existing window. Most frequently reproduced in a totally new window/tab.
3. Go to any staging URL (the screenshot is from [CHAMPVA form 10-10d on staging](https://staging.va.gov/health-care/champva/apply-form-10-10d/))

## Urgency

How urgent is this request? Please select the appropriate option below and/or provide details

- [ ] This bug is blocking work currently in progress
- [ ] This bug is affecting work currently in progress but we have a workaround
- [ ] This bug is blocking work planned within the next few sprints
- [x] This bug is not blocking any work
- [x] Other

### Details

The bug is not blocking work but it is a major a11y defect in iOS/Edge.

**Comment by artsymartha68** [comment]
Edge on iOS! I love that you were trying this on this combo. Feels like it would be very niche so I looked at the data. 

Edge on Mac comprises .08% of our traffic if I’m reading the data correctly.
Compared to Edge on Windows which is 99.88%. 

I'm finding it hard to even specify Edge on iOS.

**Comment by artsymartha68** [comment]
Found it! In the past year, 19 of 112,011,637 Edge sessions have been from iOS. That was fun to look up. Now we have a sense of the scope. 

<img width="611" alt="Screenshot 2024-04-19 at 12 04 36 PM" src="https://github.com/department-of-veterans-affairs/vets-design-system-documentation/assets/47792423/85636482-1be5-4d12-b104-76c47e2311d9">

**Comment by eli-oat** [comment]
I had a chance to dig into this a bit. There are a few things at play: 

- Edge on iOS ships with a feature on by default that will force a dark mode to any website that doesn't have one *if* the device is set to dark mode at the system level
- This is a setting that can be toggled off at a per-tab, or at the browser-level (I don't think this is a good work around, but wanted to flag for transparency)

**Comment by caw310** [comment]
DS and F team is looking into supporting dark mode so we cannot work on this until we complete that work.

**Comment by tbaker1026** [comment]
@rachaelpenfil working on dark mode this sprint - https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/4068
